src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  margin: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  color: #fff;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding-bottom: 40px;
+  }
+
+  &__description {
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-color;
+    align-content: space-evenly;
+    padding-bottom: 4px;
+    margin: 0;
+  }
+
+  &__code {
+    font-family: $font-family;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 0;
+    padding-bottom: 16px;
+    align-content: flex-start;
+  }
+
+  &__active {
+    display: flex;
+    justify-content: space-between;
+    height: 16px;
+    padding-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-family: $font-family;
```

Comment:
The variable `$font-family` is used here again, but it is not defined within this file. Ensure that it is defined in a file that is imported into this SCSS file, or define it here if it is missing.